test/lib/target-frameworks.test.ts

```diff
@@ -43,6 +43,34 @@ test('.Net .csproj dotnet-empty-manifest target framework extracted', async (t)
   t.deepEqual(targetFrameworks, [], 'targetFramework array is as expected');
 });
 
+/*
+****** fsproj ******
+*/
+
+test('.Net .fsproj simple project target framework extracted as expected', async (t) => {
+  const targetFrameworks = await extractTargetFrameworksFromFiles(
+    `${__dirname}/../fixtures/dotnet-fs-simple-project`,
+    'manifest.fsproj');
```

Comment:
this seems like an extra test?

Reply:
You're right. Messed up the "merge conflict". I'll remove the duplicate.